tox.ini

```diff
@@ -24,4 +36,4 @@ commands =
 #     H303  No wildcard (*) import.
 # Excluding our upstream submodule, and our vendored f5 configuration script.
 ignore=F403,H303
-exclude=openstack-ansible/*,f5-config.py
+exclude=rpc-openstack/*,f5-config.py
```

Comment:
Why has this been changed? openstack-ansible is the submodule and we don't want to be running flake8 against it.